harvard-lts/drs-translation-service
# Hdc 9 basic docker image

**Sets up a basic docker container for the translation service*
* * *

**GitHub Issue**: https://github.com/harvard-lts/HDC/issues/9

# What does this Pull Request do?
Creates a basic (local) Dockerfile for the app that can be modeled for the IF deployment

# How should this be tested?

1. Clone the repo
2. Checkout the branch HDC-9_basic_docker_image
3. Copy the env-template.txt to .env
4. Get the credentials from LPE Shared-HDataCommons/3a/dims ActiveMQ dev/qa transfer and process queue
5. Add the credentials from step 4 to the .env for the process and drs queues
6. Start up the docker container: `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
7. Exec into the container: `docker exec -it mqutils bash`
8. Run the command `pytest`
9. This will add a dummy message to the PROCESS_QUEUE and check that the listener retrieved it successfully.
10. Also verify that the healthcheck verifies: https://localhost:10581/healthcheck

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? Yes
